implementations/rust/ockam/ockam_vault/traits/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "ockam_vault"
+version = "0.0.0"
+authors = ["Ockam Developers"]
+edition = "2018"
+license = "Apache-2.0"
+
+[lib]
+crate-type = ["rlib"]
+
+[dependencies]
+ockam_error = {path = "../../ockam_error", version = "*"}
+ockam_macro = {path = "../../ockam_macro", version = "*"}
+zeroize = { version = "1.1", features = ["zeroize_derive"] }
+cfg-if = "1.0"
```

Comment:
do we really need this dependency?
could we use https://doc.rust-lang.org/1.49.0/std/macro.cfg.html instead?

Reply:
@mrinalwadhwa these crates are designed for different purpose. cfg-if is works exactly like standard rust cfg, but allows you to have multiple statements under one cfg. It's just for convenience and it's very lightweight (the whole lib is one macro). std::cfg does not support conditional compilation

Reply:
Great 👍